### PR TITLE
box-shadow for context-menu -> readability

### PR DESCRIPTION
**Pull request by Feichtmeier**
Fixing https://github.com/ubuntu/gtk-communitheme/issues/183#issuecomment-385289802
It's the same shadow as popovers have.

Thanks to @vinceliuice for the tip!

image

@madsrh @clobrano is this okay for you?

### CODE REVIEW

File: Communitheme/gtk-3.0/_common.scss [hunk 1]
```diff
@@ -4513,10 +4513,11 @@ decoration {
   // server-side decorations as used by mutter
   .ssd & { box-shadow: none; }
 
+  //These shadows are applied to the context-menu!
   .csd.popup & {
     border-radius: $small_radius;
-    box-shadow: 0 1px 2px transparentize(black, 0.8),
-                0 0 0 1px transparentize($_wm_border, 0.1);
+    box-shadow: 0 2px 5px transparentize(black, 0.75),
+    0 0 0 1px transparentize($_wm_border, 0.1);
```
Comment (clobrano) [inline Communitheme/gtk-3.0/_common.scss]: Could you keep the same alignment we had between the two box-shadows?

Comment (Feichtmeier) [reply Communitheme/gtk-3.0/_common.scss]: Done, thanks - should drink a coffee!

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr429`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr429` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr429`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr429` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by clobrano** [comment]
LGTM :+1:

**Pull request merged**